### PR TITLE
[4.0] Module edit. Dropdown Save&Close fails when module User Status deactivated

### PR DESCRIPTION
**Pull request by ReLater**
Pull Request for Issue https://github.com/joomla/joomla-cms/issues/24911

### Testing Instructions
- See https://github.com/joomla/joomla-cms/issues/24911 please.

### Expected and actual result
- See https://github.com/joomla/joomla-cms/issues/24911 please.

### CODE REVIEW

**Comment by infograf768** [comment]
I have tested this item :white_check_mark: successfully on 67cfd27629b90d5bf1fa9da3f6c3c2a8d244e1ef<br /><br />Note: it is also needed for site modules edit if the User Status module is unpublished.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24921">issues.joomla.org/tracker/joomla-cms/24921</a>.</sub>

**Comment by ReLater** [comment]
> Note: it is also needed for site modules edit if the User Status module is unpublished.
I'll provide a pr.

**Comment by infograf768** [comment]
We may improve this by also taking off the code from the User Status module as it will not be necessary anymore.

**Comment by infograf768** [comment]
What I meant is that adding it in the edit.php as done here solves the issue for all modules edit, site and admin.

**Comment by ReLater** [comment]
Yes, my https://github.com/joomla/joomla-cms/pull/24921#issuecomment-492995933 was nonsense ;-)

**Comment by ReLater** [comment]
> We may improve this by also taking off the code from the User Status module as it will not be necessary anymore.

Let's do it in another pr after this one has been merged. There are several modules with line
`HTMLHelper::_('bootstrap.framework');` in `tmpl/default.php` but I'm not sure if we need them for output and not just for edit purposes.

I've added a reminder to my ToDos.

**Comment by SharkyKZ** [comment]
This should probably be added to dropdown button layout.

**Comment by ReLater** [comment]
Moved fix to `layouts/joomla/toolbar/dropdown.php`

**Comment by SharkyKZ** [comment]
I have tested this item :white_check_mark: successfully on aa70678828d710292498609cc6aef94c201bcc72<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24921">issues.joomla.org/tracker/joomla-cms/24921</a>.</sub>

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on aa70678828d710292498609cc6aef94c201bcc72<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24921">issues.joomla.org/tracker/joomla-cms/24921</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24921">issues.joomla.org/tracker/joomla-cms/24921</a>.</sub>

File: layouts/joomla/toolbar/dropdown.php [hunk 1]
```diff
@@ -10,8 +10,11 @@
 defined('JPATH_BASE') or die;
 
 use Joomla\CMS\Factory;
+use Joomla\CMS\HTML\HTMLHelper;
 use Joomla\CMS\Language\Text;
 
+HTMLHelper::_('bootstrap.framework');
```
Comment (wilsonge) [inline layouts/joomla/toolbar/dropdown.php]: Can we put this inside the `if` statement please. It's only required if we have a button

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**